src/box.tsx

```diff
@@ -49,21 +50,36 @@ const Box = React.forwardRef<HTMLDivElement, Props>(
         },
         ref
     ) => {
-        const paddingSprinkles = {paddingTop, paddingBottom, paddingLeft, paddingRight};
-        let paddingStyles: React.CSSProperties = {};
-        let paddingClasses = '';
-        try {
-            paddingClasses = sprinkles(paddingSprinkles);
-        } catch (e) {
-            // if this fails, it's because the consumer passed in a value that is not a valid padding size
-            // fallback to inline styles in that case.
-            paddingStyles = {
-                paddingTop: typeof paddingTop === 'object' ? paddingTop.mobile : paddingTop,
-                paddingBottom: typeof paddingBottom === 'object' ? paddingBottom.mobile : paddingBottom,
-                paddingLeft: typeof paddingLeft === 'object' ? paddingLeft.mobile : paddingLeft,
-                paddingRight: typeof paddingRight === 'object' ? paddingRight.mobile : paddingRight,
-            };
-        }
+        const paddingTopValues =
```

Comment:
I couldn't come up with a way to use sprinkles in Box and Grid components (because I don't know how to replicate the fallback logic with pure CSS).

The only way I was able to make this work is to use CSS vars and `style()` instead of `sprinkles()` in the corresponding CSS files.